hack/remove-calico-policy/README.md

```diff
@@ -0,0 +1,13 @@
+# Disabling and removing Calico policy
+
+1. Stop and remove the calico-node Pod on each Node to prevent network policy from being (re)programmed.
+    `kubectl delete daemonset calico-node -n kube-system`
+
+2. Create a configmap with the bash script to remove the Calico iptables rules (this configmap will be used in the DaemonSet in the following step).
```

Comment:
"with the bash script" should instead say "with the script `remove-calico-policy.sh` , found in this directory"